src/sandbox/components/Worksheet.svelte

```diff
@@ -129,6 +130,15 @@
     }
     return false;
   }
+
+  function handleClipboard(action) {
+    switch (action.toLowerCase()) {
+      case "copy":  copyToClipboard(thisSheet, selection, false); break;
+      case "cut":   copyToClipboard(thisSheet, selection, true); break;
+      case "paste": pasteFromClipboard(thisSheet, selection);
+    }
```

Comment:
It would be much better to put this function in copypaste.js instead and import it from the ContextMenu instead of passing it as a prop

Reply:
Done